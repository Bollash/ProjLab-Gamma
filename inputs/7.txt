Create Space
Add Asteroid Iron
Add Settler 0
Mine 0
Status All