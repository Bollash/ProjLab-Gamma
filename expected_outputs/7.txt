Space létrehozva
A nyersanyaggal teli aszteroida hozzá lett adva a space-hez.
Sikeres volt a telepeshozzáadás
Nem lehet az aszteroidát bányászni, mert nincs átfúrva a kéreg.
Space
1
10
10
False
1
1
Asteroid
0
3
3
3
Iron
0
1
0
Settler
0
0
0
0
